docs/context/ref/Instrument.md

```diff
@@ -30,27 +30,27 @@ a property that makes it clear what value represents. Doing so will make interpr
 
 ## Schema
 
-https://fdc3.finos.org/schemas/next/instrument.schema.json
+<https://fdc3.finos.org/schemas/next/instrument.schema.json>
 
 ## Details
 
-| Property                    | Type    | Required | Example Value            | More Info                                               |
+| Property                    | Type    | Required | Example Value            | More Info                                              |
 |-----------------------------|---------|----------|--------------------------|--------------------------------------------------------|
 | `type`                      | string  | Yes      | `"fdc3.instrument"`      |                                                        |
 | `name`                      | string  | No       | `"Microsoft"`            |                                                        |
 | `id.ticker`                 | string  | No       | `"MSFT"`                 |                                                        |
-| `id.BBG`                    | string  | No       | `"MSFT:US"`              | https://www.bloomberg.com/                             |
-| `id.CUSIP`                  | string  | No       | `"594918104"`            | https://www.cusip.com/                                 |
-| `id.FDS_ID`                 | string  | No       | `"P8R3C2-R"`             | https://www.factset.com/                               |
-| `id.FIGI`                   | string  | No       | `"BBG000BPH459"`         | https://www.openfigi.com/                              |
-| `id.ISIN`                   | string  | No       | `"US5949181045"`         | https://www.isin.org/                                  |
-| `id.PERMID`                 | string  | No       | `"4295907168"`           | https://permid.org/                                    |
-| `id.RIC`                    | string  | No       | `"MSFT.OQ"`              | https://www.refinitiv.com/                             |
-| `id.SEDOL`                  | string  | No       | `"2588173"`              | https://www.lseg.com/sedol                             |
-| `market.MIC`                | string  | No       | `"XNAS"`                 | https://en.wikipedia.org/wiki/Market_Identifier_Code   |
+| `id.BBG`                    | string  | No       | `"MSFT:US"`              | <https://www.bloomberg.com/>                           |
```

Comment:
Noise in the source only, obviously the <> doesn't render, but is intended as a hint to the markdown parser. 
Rule: https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md034.md
Docusaurus' markdown parser does detect and render links and email addresses - however I have use case for the rule. I find the rule helpful in some of the docs as it either forces you to code fence the link with backticks if you don't want it to render (helpful for fully qualified appId examples which otherwise render as mailto: links or example URLs), angle brackets if you do want it to render, or the labelled link syntax. I prefer the angle bracket syntax to duplicating the link in a label in the `[label](URL)` syntax.